### PR TITLE
Add new ELF memory space for debugger

### PR DESCRIPTION
**Pull request by bmyates**
Signed-off-by: Brandon Yates <brandon.yates@intel.com>

### CODE REVIEW

**Review by jitendrasharma1989 — APPROVED** [review]

File: scripts/tools/debug.yml [hunk 1]
```diff
@@ -301,6 +301,8 @@ etors:
       value: "0"
     - name: SLM
       desc: "shared local memory space (GPU-only)"
+    - name: ELF
+      desc: "ELF file memory space"
```
Comment (wdamon-intel) [inline scripts/tools/debug.yml:305]: Add: `version: "1.10"`

**Comment by wdamon-intel** [comment]
Please change the base of this PR to oneapi-src/level-zero-spec:staging/v1.10

**Comment by aravindksg** [comment]
we need an issue to reference for this PR right?

File: scripts/tools/debug.yml (hunk 1 above)
Comment (bmyates) [reply scripts/tools/debug.yml:305]: done

**Comment by bmyates** [comment]
> Please change the base of this PR to oneapi-src/level-zero-spec:staging/v1.10

done

**Review by wdamon-intel — APPROVED** [review]

**Pull request merged**